### DISCUSSION TITLE
Ideas for Inmarsat-STD-C and NavTex

### DISCUSSION THREAD

**Discussion by mgrouch** [peterantypas/maiana#96] [Ideas]
Based on success of the design of Maiana, especially idea of putting decoder inside antenna enclosure and using ethernet
as data and a power media as only wire going down to the cabin, 
I just thought (and hoped) you could be interested in designing (following same pattern) two other devices:

one for Inmarsat-STD-C

and another for NavTex?

there are open source de-modulators / decoders for both.

and there were successful attempts to decode those transmissions with SDR and active antennas.

Decoders source codes:

NavTex:  https://arachnoid.com/JNX/JNX_source.tar.gz   (https://arachnoid.com/JNX/)

Inmarsat-STD-C: https://github.com/cropinghigh/stdcdec  (https://github.com/cropinghigh/inmarsatc)

NavTex antenna design: http://dl1dbc.net/SAQ/miniwhip.html 
The PA0RDT Mini Whip

Thanks,
--MG

**Comment by gah67** [comment]
There's a pretty simple DIY Navtex receiver detailed here [ http://www.harpoon.si/uploads/Razvoj%20podpora/Navtex.pdf | http://www.harpoon.si/uploads/Razvoj%20podpora/Navtex.pdf ] 
& one built & tested & improved here [ https://www.youtube.com/channel/UCOm-zS6cpo6cv0mkxnkiZjQ | https://www.youtube.com/channel/UCOm-zS6cpo6cv0mkxnkiZjQ ] 

that could easily (using modern sm components) be made to fit in a 1" tube 

An Open Source Inmarsat receiver would be a first and possible very popular as there is a lot of interest in it - there is a GnuRadio flow for it - [ https://thinair.aelogic.com/french-guiana-firing-exercise/ | https://thinair.aelogic.com/french-guiana-firing-exercise/ ] 
That website is the dev of Scytale-C who is working on a standalone commandline (still using an SDR) Rpi solution

**Comment by peterantypas** [comment]
A quick survey of the Navtex market shows a wide range of devices down to 180 British pounds (https://www.nasamarine.com/product/2754/). In addition, unlike with AIS, there is a large second hand market. Therefore, I will pass.

**Comment by mgrouch** [comment]
Inmarsat might be more interesting. 

Here is how antenna looks like:

https://youtu.be/PKO8hgtJUZ0

Thanks

**Reply by peterantypas** [reply]
Yeah, I've seen those. I'm not worried about antennas - I can build a proper PIFA in a snap. I'm more worried about silicon. I can't think of a low cost L-Band receiver IC with PSK support. There are plenty of sub-GHz receivers with (G)FSK support which made AIS possible.

**Reply by peterantypas** [reply]
Maybe this ... https://www.maximintegrated.com/en/products/comms/wireless-rf/MAX2121.html

**Comment by mgrouch** [comment]
It says PSK here

https://www.xonelec.it/mpn/maxim/max2112eti

**Reply by peterantypas** [reply]
It looks like this family of chips are generic receivers with baseband I/Q outputs. So in theory they could be used to decode any modulation scheme.

That said, these devices are designed for DVB-S so they expect a dish antenna with a built-in LNA. So there is definitely a need for a very low noise front end + bandpass filter. Thankfully, most GPS LNAs will work well for this. But GPS SAW filters are out of the question.

**Comment by peterantypas** [comment]
I will not build legacy satellite receivers. 

All I can say is that I have just seen the future of satellite IoT and it's less than 2 years away. When it comes, all of these services will migrate into a dirt cheap device that every boater will be able to afford. 

I hope to get started on this very soon.

**Reply by muebau** [reply]
Yeah I looked around a bit and found that there is something modern missing when it comes to systems like NAVTEX, Inmarsat-STD-C and the like.
Most of the systems use ancient technology with "human readable" communication.

**Reply by peterantypas** [reply]
There's a lot happening on this front now. We'll be seeing a new generation of constellations launching over the next year that will compete *directly* with cellular IoT in terms of economics. I'm tapping into one of these as soon as I can get silicon.

The next step is orbiting cellular towers from AST (https://ast-science.com/) and/or a satellite modem in the iPhone. That's several years away.
